plugins/source/terraform/docs/index.md

```diff
@@ -5,31 +5,24 @@ The CloudQuery Terraform plugin extracts terraform state.
 ## Install
 
 ```bash
-cloudquery init terraform
+cloudquery generate terraform
 ```
 
 ## Configuration
 
-The following configuration section can be automatically generated by `cloudquery init terraform`:
+The following configuration section can be automatically generated by `cloudquery generate terraform`:
 
 ```yaml
-providers:
-    # provider configurations
-    - name: terraform
-      configuration:
-        config:
-         - name: mylocal # local backend
-           backend: local
-           path: ./examples/terraform.tfstate
-         - name: myremote # s3 backend
-           backend: s3
-           bucket: tf-states
-           key: terraform.tfstate
-           region: us-east-1
-           role_arn: ""
-      # list of resources to fetch
-      resources:
```

Comment:
I notice the removal of `resources` from the config: is it still possible to filter the resources to fetch?

Reply:
It's not supposed to be here, it will be in the outer spec and managed by the core side of things. (It's still possible and there's no reason for it to not work)

Reply:
Oh yeah of course, makes sense 👍